inzaana_cms/database/migrations/2015_12_04_122850_create_products.php

```diff
@@ -16,7 +16,7 @@ public function up()
     {
         Schema::create('products', function (Blueprint $table) {
             $table->bigIncrements('id');
-            $table->bigInteger('user_id');
+            $table->unsignedBigInteger('user_id');
```

Comment:
why unsignedBigInt?

Reply:
same data type maintained for foreign key and primary key